config/routes.rb

```diff
@@ -67,8 +67,9 @@
   resources :news_items, except: %i[show]
   resources :reports, only: %i[index show]
 
-  resources :quotas, only: %i[new] do
+  resources :quotas do
```

Comment:
I don't think you need the resources/collection DSL if all you're doing is adding some non-restful routes.

```ruby
get '/quotas/quota_search', as: :quota_search, via: %i[get], to: 'quotas#new'
get '/search', as: :perform_search, via: %i[get post], to: 'quotas#search
```